module/VuFind/src/VuFind/View/Helper/Root/Config.php

```diff
@@ -122,4 +122,14 @@ public function alwaysDisplayIndexRecordInStaffView(): bool
         return (bool)($this->get('config')->Record
             ->alwaysDisplayIndexRecordInStaffView ?? false);
     }
+
+    /**
+     * Check if record ratings can be removed
+     *
+     * @return bool
+     */
+    public function allowsRemoveRating(): bool
```

Comment:
Might this be a better fit for VuFind\Config\AccountCapabilities rather than the generic Config helper?